cdbot/cogs/maths.py

```diff
@@ -209,6 +210,16 @@ async def latex_render(self, ctx: Context, expression: str):
                 )
                 return await ctx.send(embed=embed, delete_after=30)
 
+    @Cog.listener()
+    async def on_raw_reaction_add(self, payload):
+        if ord(str(payload.emoji)[0]) == 128465:
+            channel = await self.bot.fetch_channel(channel_id=payload.channel_id)
+            msg = await channel.fetch_message(payload.message_id)
+            filename = msg.attachments[0].filename
+            user = await self.bot.fetch_user(payload.user_id)
+            if str(user).replace("#", "") == filename[:-4]:
```

Comment:
Why replace the `#`?

Reply:
because when the file name is saved for some reason it doesn't include the # in the name so user#2843 would be user2843

Reply:
`if user.name + user.discriminator == filename[:4]:` this would work tho aswell